integration/management-api-tests/src/test/java/org/wso2/micro/integrator/api/ApiResourceTestCase.java

```diff
@@ -44,25 +39,17 @@ public void setEnvironment() throws Exception {
 
     @Test(groups = {"wso2.esb"}, description = "Test get APIs resource")
     public void retrieveAPIs() throws IOException {
-        String responsePayload = sendHttpRequestAndGetPayload(endpoint, accessToken);
-        JSONObject jsonResponse = new JSONObject(responsePayload);
-        Assert.assertEquals(jsonResponse.get(COUNT), 4, "Assert Failed due to the mismatch of " +
-                "actual vs expected resource count");
-        Assert.assertTrue(jsonResponse.get(LIST).toString().contains("AbcRestApi"), "Assert failed " +
-                "since expected resource name not found in the list");
-        Assert.assertTrue(jsonResponse.get(LIST).toString().contains("HelloRestApi"), "Assert failed " +
-                "since expected resource name not found in the list");
+        JSONObject jsonResponse = sendHttpRequestAndGetPayload(endpoint, accessToken);
+        verifyResourceCount(jsonResponse, 4);
+        verifyResourceInfo(jsonResponse, new String[]{"AbcRestApi", "HelloRestApi"});
     }
 
     @Test(groups = {"wso2.esb"}, description = "Test get APIs resource for search key")
     public void retrieveSearchedAPIs() throws IOException {
         endpoint = endpoint.concat("?searchKey=HeLloRest");
```

Comment:
Do we need to assign this to `endpoint`? If for some reason `retrieveSearchedAPIs` test runs before `retrieveAPIs` test the `retrieveAPIs` test will get failed

Reply:
can directly concat the params to the resource path. Fixed it since it can affect the test results with the order of execution, as mentioned